infrastructure_files/docker-compose.yml.tmpl

```diff
@@ -40,11 +40,11 @@ services:
       - $LETSENCRYPT_VOLUMENAME:/etc/letsencrypt:ro
       - ./management.json:/etc/netbird/management.json
     ports:
-      - $NETBIRD_MGMT_GRPC_API_PORT:33073 #gRPC port
-      - $NETBIRD_MGMT_API_PORT:33071 #API port
+      - $NETBIRD_MGMT_API_PORT:443 #API port
   #     # port and command for Let's Encrypt validation without dashboard container
   #   - 443:443
   #    command: ["--letsencrypt-domain", "$NETBIRD_DOMAIN", "--log-file", "console"]
+    command: ["--port", "443"]
```

Comment:
we need to add --log-file console back because we are overwriting the CMD from the dockerfile